msprime/trees.py

```diff
@@ -327,7 +323,10 @@ def draw(self):
         for u in self._tree.nodes():
             v = self._tree.get_parent(u)
             x = self._x_coords[u], self._y_coords[u]
-            dwg.add(dwg.circle(center=x, r=3))
+            if u in self._node_colours and self._node_colours[u] is not None:
```

Comment:
It seems not unreasonable to be able to specify that a branch or node should take the "default" colour. I thought is would be useful if that could be done either way: by omitting that node from the dict, or by including all nodes, but setting some to a value of None.